src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -55,6 +55,7 @@ type streamState int
 const (
 	STREAM_DATA_UNEXPECTED streamState = iota
 	STREAM_DATA_EXPECTED
+	STREAM_QUERY_HANDLED
```

Comment:
@nholland94 any idea why was this change introduced?

Reply:
Looks like a bad merge. I renamed these enums in a previous PR, and this is one of the old names. The only 2 enums we need here are the `STREAM_DATA_*` ones.